frontend/src/metabase/query_builder/components/data-search/SearchResults.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+import Icon from "metabase/components/Icon";
+
+import Search from "metabase/entities/search";
+
+import { SearchResultItem } from "./SearchResultItem";
+
+const MAX_SEARCH_RESULTS = 200;
+
+export function SearchResults({ searchQuery, onSelect, databaseId }) {
+  const query = {
+    q: searchQuery,
+    models: ["table", "card"],
+    limit: MAX_SEARCH_RESULTS,
+    offset: 0,
+  };
+
+  if (databaseId) {
+    query["table_db_id"] = databaseId;
+  }
+
+  return (
+    <SearchResultsRoot>
+      <Search.ListLoader query={query} wrapped reload debounced>
+        {({ list }) => {
+          if (list.length === 0) {
+            return (
+              <div className="flex flex-column align-center justify-center p4 text-medium text-centered">
+                <div className="my4">
+                  <Icon name="search" mb={1} size={32} />
+                  <h3 className="text-light">{t`No results found`}</h3>
+                </div>
+              </div>
+            );
+          }
+
+          return (
+            <ol>
+              {list.map(item => (
+                <SearchResultItem item={item} onSelect={onSelect} />
```

Comment:
don't forget a `key` here

Reply:
Oh... that reminded me to turn on that rule on `master`! Shoot...